lib/cequel/record/tasks.rb

```diff
@@ -63,12 +63,15 @@ def drop!
     puts "Dropped keyspace #{Cequel::Record.connection.name}"
   end
 
+  def models_dir_path
+    ENV['CEQUEL_MODELS_PATH'] || "#{File.expand_path('app/models', project_root)}/"
```

Comment:
I've done it, I think. :-) I set it in the "tell don't ask" style that checks for Rails::Railtie (much more reliable than Rails, as many people monkey patch the Rails module into existence in non-rails systems). If there, will use that folder. If not, will use the Dir.pwd as was used in the previous version. Then, I check the resulting pathname to see if it exists. If not, I raise a NoModelDirectoryError.